user-docs/pupil-capture.md

```diff
@@ -209,67 +204,191 @@ If you open up a session folder you will see a collection of video(s) and data f
 
 Click on the selector "Open Plugin" and select your plugin.
 
-### Pupil Groups
+#### Third-party plugins
+
+You can easily load third party plugins. Third party plugins will appear in the
+Pupil Capture or Pupil Player plugin list. Copy the plugin to the plugins folder
+within the `pupil_capture_settings` or `pupil_player_settings` folder.
+
+### Fixation Detection
+
+Fixation detectors classify fixations using dispersion and duration.
+Consider: *Dispersion* is measured as the distance between pupil positions.
+*Duration* is a specified period of time. The fixation detector plugin will
+classify pupil positions as fixations if they are within the dispersion for the
+duration of time specified.
+
+The `Fixation Detector 3D` plugin uses the eye model's 3d orientation angle to
+calculate the dispersion. The settings allow you to set the duration in seconds
+and the maximum dispersion amount in degrees.
+
+If the plugin is loaded, fixations will reduce the amount of samples that a
+[Screen Marker or Manual Marker calibration](https://docs.pupil-labs.com/master/#calibration-methods) usually requires.
+
+### Network plugins
+
+Pupil Capture has a built-in data broadcast functionality. It is based on the network library [ZeroMQ](http://zeromq.org/)
+and follows the [`PUB-SUB` pattern](http://zguide.zeromq.org/php:chapter1#Getting-the-Message-Out). Data is published with an affiliated topic.
+Clients need to subscribe to their topic of interest to receive the respective data. To save network traffic, only data
+with at least one subscription is transferred.
+
+#### Pupil Remote
+
+> {{< video-webm src="/videos/pupil-remote/pr.webm" >}}
+
+`Pupil Remote` is the plugin that functions as entry point to the broadcast infrastructure. It also provides a high level
+interface to control Pupil Capture over the network (e.g. start/stop a recording).
```

Comment:
Is there a newline between `level` and `interface`? If so, please replace with a whitespace.